### DISCUSSION TITLE
SNZB-02D

### DISCUSSION THREAD

**Discussion by JacekKac** [Koenkk/zigbee2mqtt#19493] [Devices]
Can somebody share a feedback if this device more stable than snzb-02. In case of SNZB-02 i have very bad experience with hanginup/losing network etc. sometimes snzb-02 just died and didn't responded, i switched almost 20 devices to aqara and now wodering about this one
Thanks in advance

**Comment by martin8310** [comment]
So far I have no problems with both model's.

**Reply by JacekKac** [reply]
:( iu have more than 100 SNZB-02 problems all the time:(
